src/common/dataHooks/useSongs.js

```diff
@@ -0,0 +1,70 @@
+import { useInfiniteQuery, useQuery } from '@tanstack/react-query'
+import { useParams } from 'react-router-dom'
+
+// FPCC
+import api from 'services/api'
+import { SONGS, TYPE_SONG } from 'common/constants'
+
+const formatSong = (result) => ({
+  acknowledgement: result?.acknowledgements || [],
+  audio: result?.relatedAudio || [],
+  coverVisual: result?.coverImage || {},
```

Comment:
Just noting here that we discussed that I would update this one when I merge the fv-be changes